muse/components/load_balancer.py

```diff
@@ -201,13 +201,13 @@ async def update_servers(servers: List[str]):
 
         @app.post("/api/surprise-me")
         async def surprise_me():
-            data = Data(dream=random_prompt())
+            data = Data(prompt=random_prompt())
```

Comment:
@aniketmaurya, this change wasn't there until my rebase, but it looks like it's needed there.